contracts/signature_schemas/signatureSchemas.js

```diff
@@ -232,6 +236,22 @@ schemas.updateReplicaSet = [
   { name: 'nonce', type: 'bytes32' }
 ]
 
+schemas.manageUser = [
+  { name: 'userId', type: 'uint' },
+  { name: 'action', type: 'string' },
+  { name: 'metadata', type: 'string'},
+  { name: 'nonce', type: 'bytes32'}
+]
+
+schemas.manageEntity = [
+  { name: 'userId', type: 'uint'},
```

Comment:
is it work making id type a string now? just a thought. probably not worth changing today, but i guess with zero chain state this becomes quite easy

Reply:
are you referring to entityId? I can definitely handle it now

Reply:
referencing this this - https://github.com/AudiusProject/audius-protocol/pull/3234#discussion_r899545019